src/pages/Cart/Cart.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import "./cart.css";
+import { Footer, Navbar } from "../../components";
+import { Cardcart } from "./component/Cardcart";
+import { image7 } from "../../assets/images";
+import { Cartbill } from "./component/Cartbill";
+
+export const Cart = () => {
+  return (
+    <div className="e_container">
+      <Navbar auth={"Login"} />
+      <div>
+        <h1 className="headline">My Cart</h1>
+        <div className="title_underline"></div>
+      </div>
+      <section className="cart_container dis_flex">
+        <Cardcart
+          cardtitle={"SOM2SOM Women Solid Straight Kurta"}
+          cartimg={image7}
+          cartprice={"Rs.250"}
+          cutprice={"Rs.500"}
+          discount={"50%"}
+          Quantity={"Quantity :"}
```

Comment:
I think you can destructure it and pass single value through data. It will convenient.

Reply:
it's just for a dummy card I will change this thank you for pointing out.